src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/RuntimeBroker.cs

```diff
@@ -552,6 +564,37 @@ public async Task RemoveAccountAsync(ApplicationConfiguration appConfig, IAccoun
                 }
             }
         }
+        public Dictionary<string, string> GetSsoPolicyHeaders()
+        {
+            using LogEventWrapper logEventWrapper = new LogEventWrapper(this);
+            Debug.Assert(s_lazyCore.Value != null, "Should not call this API if MSAL runtime init failed");
+
+            NativeInterop.SsoPolicy ssoPolicy = new SsoPolicy();
+            _logger.Info(() => $"[RuntimeBroker] Broker returned SsoPolicyType {ssoPolicy._ssoPolicyType} and errorCode {ssoPolicy._errorCode}.");
+
+            var ssoPolicyHeaders = new Dictionary<string, string>();
```

Comment:
>= new Dictionary<string, string>(); [](http://example.com/codeflow?start=33&length=35)

Out side of EU will be always allowed. 
So we should just call new when we need to.

Reply:
not critical now. can look into this later